palantirnet/search_api_federated_solr
# UMUS-77: Backport federated fields

See [UMUS-77](https://palantir.atlassian.net/browse/UMUS-77).

### Description

Backports the "federated field" functionality from the d8 version of the module, and adds a new processor that allows remapping the field names in the Drupal 7 Search API index.

### Testing instructions

1. Check out the `backporting-UMUS-77-UMUS-88` branch of the uofmhealth site:
  ``` 
  cd web/uofmhealth/
  git fetch --all
  git checkout backporting-UMUS-77-UMUS-88
  ```
2. Follow the instructions on the PR for that branch: https://github.com/palantirnet/uofmhealth/pull/20
3. Make sure the cardinality of the fields matches the schema.


### Remaining tasks

- [ ] wait 'till UMUS-76 is in, or combine this with it
